sys/auto_init/auto_init.c

```diff
@@ -20,8 +20,8 @@
 
 #include "auto_init.h"
 
-#ifdef MODULE_SHT11
-#include "sht11.h"
+#ifdef MODULE_SHT1X
+void auto_init_sht1x(void);
```

Comment:
No need to declare this function here. Just use `extern` before the call to the function.